pkg/domain/infra/abi/manifest.go

```diff
@@ -90,18 +90,25 @@ func (ir *ImageEngine) ManifestInspect(ctx context.Context, name string) ([]byte
 			continue
 		}
 
-		if !manifest.MIMETypeIsMultiImage(manifestType) {
-			appendErr(errors.Errorf("manifest is of type %s (not a list type)", manifestType))
-			continue
-		}
 		result = manifestBytes
 		manType = manifestType
 		break
 	}
 	if len(result) == 0 && latestErr != nil {
 		return nil, latestErr
 	}
-	if manType != manifest.DockerV2ListMediaType {
+
+	switch manType {
+	case manifest.DockerV2Schema2MediaType:
+		fmt.Printf("Warning! The manifest type %s is not a manifest list.  Displaying the the manifest of the single image.\n", manType)
```

Comment:
```suggestion
		fmt.Fprintf(os.Stderr, "Warning! The manifest type %s is not a manifest list.  Displaying the the manifest of the single image.\n", manType)
```
I know this has been bashed around a bit but errors should go to stderr  IMHO

Reply:
Thanks. Fixed.